src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java

```diff
@@ -65,18 +64,21 @@
   @Nullable
   private final Tracer tracer;
 
+  private final Set<String> headerTags;
```

Comment:
nit: this should nullable if the argument to the constructor is. Or we can probably remove the argument as nullable.

Reply:
nvm, this headerTags are not nullable in constructor too.